### ISSUE TITLE
An Application Error occured in Subtitle Edit 3.5.16.0 - Index was outside the bounds of the array.

### ISSUE THREAD

**Issue by FeMaster1** [SubtitleEdit/subtitleedit#4397]
Error box indicated that I should report the error here. Can't copy and paste from the error box unfortunately, and I'm not typing it all out by hand, so a screen capture of it is below. I was doing a spell check of some SSA subtitles embeded in an H265 .MKV video that had a LOT of errors. Not sure if that had anything to do with it or not.

![Subtitle Edit Error](https://user-images.githubusercontent.com/38301166/94628331-5aab1700-0285-11eb-8bf3-08b9dd4864ff.jpg)

**Comment by niksedk** [comment]
@FeMaster1: thx for the info - could you zip the subtitle file and attach it here?

(I think you can copy and paste from the error box by pressing ctrl+c)

**Comment by FeMaster1** [comment]
Will see what I can do. I already had stripped it out of the video and tossed it, as I deemed it just too much work to fix. It was BAD! I should be able to grab another copy and then I'll pull it out of the video file and attach it.

**Comment by FeMaster1** [comment]
Subtitle file extracted and attached.
[English - (Application Error).zip](https://github.com/SubtitleEdit/subtitleedit/files/5308633/English.-.Application.Error.zip)

**Comment by niksedk** [comment]
Thx for the file - skipping through all 1580 lines I did not get any errors - what line number caused the crash and what did you do in that line?
(yeah, a really BAD subtitle!)

**Comment by FeMaster1** [comment]
I was making an attempt to actually go through and correct the atrocious errors in it, unfortunately I don't know what line caused the problem. I was maybe about 10% to 15% of the way through the subtitles using the Spell Check function. I used quite a few of the "Change All" for my manual edits, and whole lot of "Always Use" for the corrections that showed up in the suggestion box on the right. I'm wondering if maybe I somehow over-exerted the Always Use function?

I was able to click the retry button, which briefly allowed my to continue on before another error box came up. I did this a total of 4 times, before I finally happened to notice how far into the subtitles I was, and just said the hell with it and gave up on them. It would have taken me hours and hours to correct that mess. I didn't pay attention to what exactly was in the subsequent error boxes. In my brief view of them, which just long enough to find and click the retry button, they looked basically the same with about as much text in each, so I'm guessing they were all similar errors.
